### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jamen**
I would like to fix some bugs before we merge this, so that what is documented works.

We can also reivew writing errors, things we need to add, etc. on this PR in the meantime.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,73 +1,87 @@
...
 cordlr
 ```
 
-But, this is a bot that is based around [plugins](plugins.md).  When you first get cordlr, it probably won't do anything because you haven't installed or loaded any plugins yet.  You will see this message:
+This starts the bot.  But, you will most likely see this:
 
-> Loaded bot (with no plugins)
+```sh
+Error: No plugins
+```
```
Comment (jamen) [inline docs/getting-started.md:13]: See #86

**Comment by jamen** [comment]
@mees- this can be merged now too.  We can fix errors as we find them.

**Comment by jamen** [comment]
Oh wait, merge conflicts.  Let me fix

**Comment by jamen** [comment]
fixed, lets squash this just so it is a clean history lol

**Review by mees- — APPROVED** [review]

**Pull request merged**